api/proto/hashedrekord.proto

```diff
@@ -23,11 +23,16 @@ import "verifier.proto";
 option go_package = "github.com/sigstore/rekor-tiles/pkg/generated/protobuf";
 
 // A request to add a hashedrekord to the log
-message HashedRekordRequest {
-    // A single signature over the hashed data
-    bytes signature = 1 [(google.api.field_behavior) = REQUIRED];
+message HashedRekordRequestV0_0_2 {
     // The hashed data
-    dev.sigstore.common.v1.HashOutput data = 2 [(google.api.field_behavior) = REQUIRED];
-    // A key/cerificate to verify the signature
-    Verifier verifier = 3 [(google.api.field_behavior) = REQUIRED];
+    dev.sigstore.common.v1.HashOutput data = 1 [(google.api.field_behavior) = REQUIRED];
+    // A single signature over the hashed data with the verifier needed to validate it
+    SignatureAndVerifier signature = 2 [(google.api.field_behavior) = REQUIRED];
```

Comment:
Do we want to rename `signature` since it also includes a verifier, here and in the entry?

Reply:
yeah maybe, our v1 api used
```
signature: {
   content:...
   public_key: {
      content:...
   }
}
```

but yeah we could rename this to `signatureAndVerifier`

Reply:
actually I think we can just structure this like
```
signature: {
   content:...
   verifier: {
      content:...
   }
}
```

Reply:
I’d be good with that too, I like that over a longer name.